### PR TITLE
[docs] add shell completion instruction

### PR DESCRIPTION
**Pull request by lalyos**
Signed-off-by: Lajos Papp <lalyos@yahoo.com>

### CODE REVIEW

**Comment by cjellick** [comment]
I seem autocompletion working without that command. Wonder if installing from homebrew does that automatically

**Comment by lalyos** [comment]
Ohh, you are right, I've used the `curl https://get.acorn.io | sh` method.
Maybe I can move this section under the [curl|sh install](https://docs.acorn.io/installation/installing#curlsh-install-macos--linux) section?

File: docs/docs/30-installation/01-installing.md [hunk 1]
```diff
@@ -72,6 +72,21 @@ The last successful build from the HEAD of the main branch is available for
 [Linux](https://cdn.acrn.io/cli/default_linux_amd64_v1/acorn), and
 [Windows](https://cdn.acrn.io/cli/default_windows_amd64_v1/acorn.exe)
 
+### Shell completion
+
+For best developer experience, shell autocompletion is provided, but the acorn cli subcommand is hidden.
```
Comment (cjellick) [inline docs/docs/30-installation/01-installing.md:77]:
```suggestion
The Acorn CLI supports command autocompletion. If you installed acorn using homebrew, this is already configured for you. If you installed using the manual or curl|sh method, you must enable shell completion yourself.

```

File: docs/docs/30-installation/01-installing.md [hunk 2]
```diff
@@ -72,6 +72,21 @@ The last successful build from the HEAD of the main branch is available for
...
+To set autocompletion for the actual terminal session, use the matching shell:
```
Comment (cjellick) [inline docs/docs/30-installation/01-installing.md:78]:
```suggestion
To set autocompletion for the current terminal session, use the command that matches your shell:
```

**Comment by cjellick** [comment]
I threw in a some clarification. If you agree to them, you can "sign off and commit" them and then we can merge.

**Comment by ibuildthecloud** [comment]
@lalyos I'm glad to see you around, look forward to your feedback :)

**Review by cjellick — APPROVED** [review]

**Pull request merged**